wirexfers/providers/ipizza.py

```diff
@@ -96,19 +102,25 @@ def parse_response(self, form, success=True):
         if not PKCS1_v1_5.new(self.keychain.public_key) \
                          .verify(SHA.new(m), b64decode(f('MAC'))):
             raise InvalidResponseError
+
+        timestamp = datetime.strptime(f('T_DATETIME'), '%Y-%m-%dT%H:%M:%S+0300')
+        now = datetime.now()
+        if timestamp > (now + timedelta(seconds=300)) or timestamp < (now - timedelta(seconds=300)):
+            raise Exception('Response outdated.')
```

Comment:
`datetime.now()` returns local time without timezone information.

We probably have to add `pytz` as dependency and use `datetime.now(pytz.utc)` instead.